awesome_dashboard/static/src/dashboard/pie_chart/pie_chart.js

```diff
@@ -0,0 +1,37 @@
+import { Component, onMounted, onWillUnmount, onWillStart, useEffect, useRef } from "@odoo/owl";
+import { loadJS } from "@web/core/assets";
+
+export class PieChart extends Component {
+    static template = "awesome_dashboard.pie_chart";
```

Comment:
Template names are camelCase instead of snake_case
```suggestion
    static template = "awesome_dashboard.PieChart";
```

Reply:
TIL it's something else called PascalCase